### DISCUSSION TITLE
Renovate fails to update maven property version in multi-module maven project

### DISCUSSION THREAD

**Discussion by jamesmmchugh** [renovatebot/renovate#22171] [Suggest an Idea]
**What Renovate type are you using?**
N/A

**Describe the bug**
GIVEN: If you have a multi-module maven project, where the parent (or any module that is higher in the dependency tree) module defines the artefact version as a property but doesn't use the property itself, and the downstream module inherits the property and uses it.
THEN: Renovate doesn't update the dependencies in the downstream module.

**Did you see anything helpful in debug logs?**
N/A

**Expected behavior**
Renovate should update the property in the parent/upstream module.

**Screenshots**
N/A

**Additional context**
N/A

**Comment by rarkins** [comment]
Any public example or reproduction available?

**Comment by rarkins** [comment]
@zharinov I thought we handled this case already?

**Comment by zharinov** [comment]
Yes, it should handle it, so most probably there is a bug

**Comment by rarkins** [comment]
In that case let’s wait for a reproduction or more details, if we already have known working cases too

**Comment by rarkins** [comment]
@jamesmmchugh can you provide any more details?
